src/components/Counter.js

```diff
@@ -20,10 +20,10 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.increment()}>
```

Comment:
Instead of using an anonymous function here, you can pass in the function directly by doing `onClick={this.props.increment}`